src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,306 @@
+            description.Append(Environment.NewLine);
+            description.Append("Action : " + repositoryBranchUpdateHistory.Action);
+            description.Append(Environment.NewLine);
+            description.Append("/FYI : " + fyiHandles);
+            description.Append(Environment.NewLine + Environment.NewLine + Environment.NewLine);
+
+            // Get repo info used for create/ update gitHub issue
+            Octokit.Repository repo = await client.Repository.Get(_options.Owner, _options.Repository);
+            try
+            {
+                using (ITransaction tx = _stateManager.CreateTransaction())
+                {
+                    if (await gitHubIssueEvaluator.ContainsKeyAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch)))
+                    {
+                        // issue exists so just update the issue
+                        var issueNumber = await gitHubIssueEvaluator.TryGetValueAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch));
+                        Logger.LogInformation("Updating a gitHub issue number : " + issueNumber + " for the error : " + repositoryBranchUpdateHistory.ErrorMessage +
+                            " for the repository : " + repositoryBranchUpdateHistory.Repository);
+                        if (issueNumber.HasValue)
+                        {
+                            try
+                            {
+                                var issue = await client.Issue.Get(_options.Owner, _options.Repository, issueNumber.Value);
+                                var issueUpdate = issue.ToUpdate();
+                                issueUpdate.Body = issue.Body + Environment.NewLine 
+                                    + Environment.NewLine + description.ToString();
+                                issueUpdate.State = ItemState.Open;
+                                await client.Issue.Update(_options.Owner ,_options.Repository, issueNumber.Value, issueUpdate);
+                            }
+                            catch(Exception ex)
+                            {
+                                Logger.LogError(ex.Message,"Unable to update issue number " + issueNumber + " in GitHub.");
+                            }
+                        }
+                        else
+                        {
+                            Logger.LogInformation("Unable to retrive the issuenumber from the reliable services.");
+                        }
+                    }
+                    else
+                    {
+                        string labelName = "DependencyUpdateError";
+                        Logger.LogInformation("Creating a gitHub issue for the error : " + repositoryBranchUpdateHistory.ErrorMessage +
+                            " for the repository : " + repositoryBranchUpdateHistory.Repository);
+                        var createIssue = new NewIssue("Dependency Update Error");
```

Comment:
This is a _really_ generic title. Can't we at least add the repository and branch to it?

Reply:
added repo and branch to the title.  I initially added this to the title and it was a lot of text so I removed it. Now I added it back

Reply:
is the "repository" that god awful full URL?  Could we just take like the last piece of it? So turn "https://github.com/dotnet/corefx" into just "corefx"? That should result in a more compact title.